deploy/operator/apply/01_FabricGatewayCRD.yaml

```diff
@@ -82,6 +82,33 @@ spec:
                   will be able to access the API even if they match all other authorization requirements. If the whitelist
                   is empty, then no service tokens can access via the gateway
                 example: stups_some-service-name
+            x-fabric-cors-support:
+              type: object
+              description: |
+                This section configures Cross-Origin Resource Sharing (CORS) support in the gateway. An HTTP OPTIONS endpoint will be created for each path,
+                fulfilling the preflight requirement for CORS. Currently the preflight request is the only part of CORS we support, we do not add the allowed origins
+                header to all other endpoints, so this will need to be done in your service.
+              required:
+                - state
```

Comment:
Does `state` make sense as a field here? In what scenario do we add the cors support object but mark it as disabled?

Reply:
I added it because it was in the solution proposal. It also seemed to me it might be useful to be able to disable it without removing the config, but if there's no clear use case I'll remove it